.github/workflows/ci.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 22.x]
+        node-version: [22.x]
```

Comment:
jest no longer supports node 12  https://jestjs.io/docs/upgrading-to-jest29

https://github.com/cesine/rickshaw/actions/runs/12040292801/job/33569879332

```
Run npm run coverage

> rickshaw@1.7.1 coverage /home/runner/work/rickshaw/rickshaw
> jest --coverage

/home/runner/work/rickshaw/rickshaw/node_modules/jest/node_modules/jest-cli/build/run.js:135
    if (error?.stack) {
              ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:[9](https://github.com/cesine/rickshaw/actions/runs/12040317722/job/33569956048#step:6:10)15:16)
```